README.md

```diff
@@ -10,6 +10,13 @@ examples get run and tailor the output.
     gem install rspec-core # for rspec-core only
     rspec --help
 
+  Want to run against the `master` branch? You'll need to include the dependent
+  RSpec repos as well. Add the following to your `Gemfile`:
+
+    %w[rspec rspec-core rspec-expectations rspec-mocks rspec-support].each do |lib|
```

Comment:
Should this be wrapped in triple backtricks (with a `ruby` language specifier) so it's syntax highlighted?